src/main/java/it/gov/pagopa/pu/debtpositions/model/view/debtpositiontypeorg/DebtPositionTypeOrgWithActiveOrganizationsCount.java

```diff
@@ -0,0 +1,24 @@
+package it.gov.pagopa.pu.debtpositions.model.view.debtpositiontypeorg;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.io.Serializable;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.Formula;
+
+@Entity
+@Table(name = "debt_position_type_org")
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class DebtPositionTypeOrgWithActiveOrganizationsCount implements Serializable {
```

Comment:
you are not checking the status of the organization
```suggestion
public class DebtPositionTypeOrgCountByOrganizationId implements Serializable {
```